plotpal/volumes.py

```diff
@@ -647,6 +659,16 @@ def plot_colormesh(self, dsets, ni, pl, **kwargs):
         if self.divide_radial_stdev:
             shell_field /= self.radial_stdev_func(self.r_outer)
         self.out_data['field'] = np.pad(shell_field, ((0,1), (0,1)), mode = 'edge')
+        
+        
+        # Build inner shell field
+        shell_field = dsets[self.inner_shell][ni].squeeze()
```

Comment:
I think this code block will throw an error if self.inner_shell is None; let's add an if-statement and indent here.